hello.py

```diff
@@ -248,11 +281,49 @@ def check_ds():
         err_file.close()
         log2_file.close()
         err2_file.close()
-
-    
-                
-
     return json.dumps(fileNames)
+
+@app.route('/das_driver_single', methods=['GET','POST'])
+def das_driver_single():    
+    """
+    Same as above, just does it with single dataset, could refactor to use the same function, but maybe laterz
+    """
+    data = json.loads(request.get_data())
+    dsName = data['_id']
+    __release = data['CMSSW']
+    os.chdir(WORK_DIR + "/Test_Folder/"+__release +"/src")
+    fileNames = {}
+    fileNames[dsName] = {}
+    log_file = file('dasTest_outSingle.txt','w+')
+    err_file = file('dasTest_errSingle.txt','w+')
+    log2_file = file('dasTest_2outSingle.txt','w+')
+    err2_file = file('dasTest_2errSingle.txt','w+')
+    proc = subprocess.call("eval `scramv1 runtime -sh`; cat /afs/cern.ch/user/j/jsiderav/private/PdmVService.txt | voms-proxy-init -voms cms -pwstdin > dasTest_voms.txt;export X509_USER_PROXY=$(voms-proxy-info --path); das_client.py  --limit 0 --query 'site dataset="+dsName+"' --key=$X509_USER_PROXY --cert=$X509_USER_PROXY --format=json",stdout=log_file,stderr=err_file,shell=True)
```

Comment:
need changes to use service account. most likely to have grid proxy path in config.